packages/EntityBackend/routes/discrepanciesRouter.js

```diff
@@ -0,0 +1,143 @@
+// import modules
+const express = require("express");
+const router = express.Router();
+const axios = require("axios");
+
+// db setup
+const DbConnection = require("../db");
+
+//Used to get a aggregated discrepancy report for an entity
+router.get("/:companyId/:borrowerId/report", async (req, res) => {
+  try {
+    /**
+     * companyId : used to identify which company this report belongs to
+     * BorrowerId : used to identify the entity across different external sources
+     */
+    const companyId = req.params.companyId;
+    const BorrowerId = req.params.borrowerId;
+
+    /** Using this information, we would know which custom api calls to dispatch for a discrepancy report. */
+    let entityConfigurations = axios({
+      method: "GET",
+      url: `http://localhost:4005/config/${companyId}`,
+    });
+
+    let configuredApiCalls = await entityConfigurations.then((response) => {
+      let entityConfigs = response.data.entityConfiguration;
+      return entityConfigs;
+    });
+
+    /** resultWithMapping : would be the final output after aggregating/mapping all the data from multiple external sources  */
+    let resultWithMapping = [];
+
+    /** list of all new keys used in resultWithMapping */
+    let allNewMappedKeys = {};
+
+    /** list of all External Sources that was used */
```

Comment:
this will be a list of all external systems used and field name after looping through all calls